homeassistant/components/tuya/models.py

```diff
@@ -391,44 +436,3 @@ def find_dpcode(
                 return type_information
 
     return None
-
-
-class ComplexValue:
-    """Complex value (for JSON/RAW parsing)."""
-
-    @classmethod
-    def from_json(cls, data: str) -> Self:
-        """Load JSON string and return a ComplexValue object."""
-        raise NotImplementedError("from_json is not implemented for this type")
-
-    @classmethod
-    def from_raw(cls, data: str) -> Self | None:
-        """Decode base64 string and return a ComplexValue object."""
-        raise NotImplementedError("from_raw is not implemented for this type")
-
-
-@dataclass
-class ElectricityValue(ComplexValue):
-    """Electricity complex value."""
```

Comment:
This has been replaced by 6 wrapper classes:
- current from json
- power from json
- voltage from json
- current from binary
- power from binary
- voltage from binary

This allows us to control better the native unit and the suggested unit